Sources/Testing/Testing.docc/DefiningTests.md

```diff
@@ -87,7 +87,7 @@ a test is unable to run due to limited availability:
 
 ```swift
 @available(macOS 11.0, *)
-@available(swift, introduced: 5.9, message: "Requires Swift 5.9 features to run")
+@available(swift, introduced: 8.0, message: "Requires Swift 8.0 features to run")
```

Comment:
Is "8.0" intentional? My first thought is "6.0".

Also why bother to updating the example here? The original "5.9" seems fine to me.

Reply:
8.0 is intentional as an arbitrary Swift version that doesn't exist. 5.9 doesn't make sense because all swift-testing tests require at least 5.10 to run, so availability limited to 5.9 doesn't do anything.